recipes/aim-ui/meta.yaml

```diff
@@ -38,3 +38,4 @@ about:
 extra:
   recipe-maintainers:
     - hadim
+    - gorarakelyan
```

Comment:
See https://github.com/conda-forge/staged-recipes/pull/21653#issuecomment-1371489453 for approval